src/TaskList.svelte

```diff
@@ -78,14 +78,18 @@ export let scrollingDuration = 2000;
 
   .entry {
     display: flex;
+    animation-name: fadeIn;
```

Comment:
you could apply those values if `:not(.entry-done):not(.entry-highlight)` that way you don't need to apply `!important`